app/controllers/works_controller.rb

```diff
@@ -29,7 +29,8 @@ def viewer_images_info
   private
 
   def decorator
-    @decorator ||= if @work.is_oral_history? && has_oh_audio_member? && @work.oral_history_content&.available_by_request_off?
+    byebug
```

Comment:
Stray `byebug`.

Reply:
good catch thank you

If this passed tests, I guess tests didn't exersize this method, which is kind of weird too! Maybe I'll see if I can add a test here too.

Reply:
Hmm, no, the test does exersize this method. I wonder how it passed CI?

Reply:
If I look at CI, and look at the log... there are a bunch of byebug prompts in the logs! I wonder how the tests pass anyway, maybe after so many seconds `byebug` prompt automatically continues?

Would probably be good to make it raise in tests... but I intentionally use byebug in tests all the time... could make it raise in tests only when run on CI, there is an ENV variable github actions sets or could set. 

But for now I'll just remove the 'byebug.

Reply:
Makes sense. Another way to approach this would be a github pre-commit hook that rejects commits with that string in them. I can't imagine a legitimate scenario in which we would ever want that string in our codebase.